src/main.rs

```diff
@@ -57,8 +67,9 @@ async fn main() -> Result<(), ExitFailure>{
         location = args[1].clone();
     }
     println!("{}",location);
-    let res = WeatherResponse::get(&api_key,&location).await?;
-    println!("{}'s current temperature: {:?}", location, res);
-    println!("{}", location);
+    // let res = WeatherResponse::get(&api_key,&location).await?;
+    // println!("{}'s current temperature: {:?}", location, res);
+    let forecast = WeatherResponse::get_today(&api_key,&location).await?;
+    println!("{}'s forecast: {:?}", location, forecast);
```

Comment:
maybe for exercise implement the display trait for  `WeatherResponse` (you might need to add location to the WeatherResponse, but I think that's ok)